### ISSUE TITLE
Error Unable to generate route for page ID 

### ISSUE THREAD

**Issue by denniserdmann** [terminal42/contao-changelanguage#258]
Changelanguage: 3.7.7
Contao: 4.13.47

Stack Trace:
```
Contao\CoreBundle\Exception\RouteParametersException:
Unable to generate route for page ID 174.

  at vendor/contao/core-bundle/src/Routing/PageUrlGenerator.php:110
  at Contao\CoreBundle\Routing\PageUrlGenerator->generate('page_routing_object', array('parameters' => '/'), 0)
     (vendor/symfony-cmf/routing/src/DynamicRouter.php:180)
  at Symfony\Cmf\Component\Routing\DynamicRouter->generate('page_routing_object', array('_content' => object(PageModel), 'parameters' => '/'), 0)
     (vendor/symfony-cmf/routing/src/ChainRouter.php:248)
  at Symfony\Cmf\Component\Routing\ChainRouter->generate('page_routing_object', array('_content' => object(PageModel), 'parameters' => '/'), 0)
     (vendor/contao/core-bundle/src/Resources/contao/models/PageModel.php:1415)
  at Contao\PageModel->getAbsoluteUrl('/')
     (vendor/terminal42/contao-changelanguage/src/Navigation/NavigationItem.php:173)
  at Terminal42\ChangeLanguage\Navigation\NavigationItem->getHref(object(UrlParameterBag))
     (vendor/terminal42/contao-changelanguage/src/Navigation/NavigationItem.php:185)
  at Terminal42\ChangeLanguage\Navigation\NavigationItem->getHref(object(UrlParameterBag), true)
     (vendor/terminal42/contao-changelanguage/src/FrontendModule/ChangeLanguageModule.php:157)
  at Terminal42\ChangeLanguage\FrontendModule\ChangeLanguageModule->generateTemplateArray(object(NavigationItem), object(UrlParameterBag))
     (vendor/terminal42/contao-changelanguage/src/FrontendModule/ChangeLanguageModule.php:123)
  at Terminal42\ChangeLanguage\FrontendModule\ChangeLanguageModule->compile()
     (vendor/contao/core-bundle/src/Resources/contao/modules/Module.php:214)
  at Contao\Module->generate()
     (vendor/terminal42/contao-changelanguage/src/FrontendModule/ChangeLanguageModule.php:67)
  at Terminal42\ChangeLanguage\FrontendModule\ChangeLanguageModule->generate()
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:442)
  at Contao\Controller::getFrontendModule(object(ModuleModel), 'header')
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:194)
  at Contao\PageRegular->prepare(object(PageModel))
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:61)
  at Contao\PageRegular->getResponse(object(PageModel), true)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/FrontendIndex.php:320)
  at Contao\FrontendIndex->renderPage(object(PageModel))
     (vendor/symfony/http-kernel/HttpKernel.php:163)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:75)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/preview.php:46)   
     ```

**Comment by fritzmg** [comment]
You should always provide more information and not post a stack trace largely without any further comments.

**Comment by lukasbableck** [comment]
I can confirm this also affects Contao 5.3.13.

**Comment by aschempp** [comment]
> You should always provide more information and not post a stack trace largely without any further comments.

This has already been discussed in Slack, the stack trace (the ticket actually) is more than enough ;) Alread on to fixing this.

**Comment by aschempp** [comment]
Fixed in 6617862bc236d24c54aefa4b45519dd670741698
